producers-commissioning/src/end-to-end-test/java/uk/gov/moj/sdt/producers/comx/SubmitBulkTest.java

```diff
@@ -45,60 +14,53 @@
 
 /**
  * Test class for end to end web service tests..
- * 
+ *
  * @author Robin Compston
- * 
  */
-@RunWith (SpringJUnit4ClassRunner.class)
-@ContextConfiguration (locations = {"classpath*:/uk/gov/moj/sdt/producers/comx/spring*e2e.test.xml",
-        "classpath*:/uk/gov/moj/sdt/utils/**/spring*.xml", 
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(locations = {"classpath*:/uk/gov/moj/sdt/producers/comx/spring*e2e.test.xml",

```

Comment:
Should this be converted to annotation based configuration?

Reply:
Will be addressed in separate ticket: [SDT-128](https://tools.hmcts.net/jira/browse/SDT-128)